test/lit/passes/asyncify_mod-asyncify-always-and-only-unwind_O.wast

```diff
@@ -41,7 +41,16 @@
 
   ;; CHECK:      (func $calls-import (; has Stack IR ;)
   ;; CHECK-NEXT:  (local $0 i32)
-  ;; CHECK-NEXT:  (call $import)
+  ;; CHECK-NEXT:  (local.set $0
+  ;; CHECK-NEXT:   (block $__asyncify_unwind (result i32)
+  ;; CHECK-NEXT:    (block
+  ;; CHECK-NEXT:     (call $import)
+  ;; CHECK-NEXT:     (br $__asyncify_unwind
+  ;; CHECK-NEXT:      (i32.const 0)
+  ;; CHECK-NEXT:     )
+  ;; CHECK-NEXT:    )
+  ;; CHECK-NEXT:   )
+  ;; CHECK-NEXT:  )
```

Comment:
Why did this change?

Reply:
In old code, after some optimization passes, this happened:
```wast
(select
  ...
  ...
  (global.get $__asyncify_state)
)
```
Because this code is checking whether the current state is normal (0) or rewinding (2), this can be optimized by https://github.com/WebAssembly/binaryen/blob/0fe08192d866f72d0f4e680c862bb7af48dec1ac/src/passes/Asyncify.cpp#L1925-L1937

But now generated code is different and after optimization passes this ended up like this:
```wast
(i32.eqz                                                                   
  (global.get $__asyncify_state)                                            
)
``` 
And we didn't have the same optimization for `Unary`. So added a similar optimization for `i32.eqz`, so that the whole `i32.eqz` can be replaced with `(i32.const 0)`.